tests/NHSD.BuyingCatalouge.Ordering.Api.Testing.Data/Entities/OrderItemEntity.cs

```diff
@@ -84,7 +84,8 @@ public sealed class OrderItemEntity : EntityBase
                 @Price,
                 @Created,
                 @LastUpdated
-            );";
+            );
+            SELECT SCOPE_IDENTITY();";
```

Comment:
Is there a reason why you have added this to the tests, just cant see where it is used to get the ID